apps/public-docsite-resources/package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "build": "just-scripts build",
     "bundle": "just-scripts bundle",
-    "lint": "just-scripts lint",
```

Comment:
this is the only project in whole monorepo that uses package API within its just-config, thus when `just lint` is executed it will parse the imports and explode as that package has not been build. with this gone we can make `lint` standalone task thus improving CI pipeline speeds and complexity of our codebase 🙌